crates/cdk/Cargo.toml

```diff
@@ -30,6 +30,7 @@ sysinfo = { workspace = true, default-features = false }
 tempfile = { workspace = true }
 toml = { workspace = true, features = ["parse", "display", "preserve_order"] }
 tracing = { workspace = true }
+openssl = { workspace = true, features = ["vendored"] } # cargo-generate requires openssl
```

Comment:
shouldn't this be using fluvio future's openssl dep?

Reply:
cdk needs openssl only for `cargo-generate`, we can also enable the feature at fluvio future.